### PR TITLE
PR pipeline: Ubuntu compile and ICW w/ ORCA

### PR DESCRIPTION
**Pull request by goutamtadi1**
- Adding Ubuntu platform check for PR pipeline

### CODE REVIEW

File: concourse/pipelines/pr_pipeline.yml [hunk 1]
```diff
@@ -88,6 +95,17 @@ jobs:
           TARGET_OS_VERSION: 6
           BLD_TARGETS: "clients loaders"
 
+      - task: compile_gpdb_and_orca_ubuntu16
```
Comment (kmacoskey) [inline concourse/pipelines/pr_pipeline.yml]: Should this be renamed to match the same job in the other pipelines?`compile_gpdb_and_orca_conan_ubuntu16`
https://github.com/greenplum-db/gpdb/blob/master/concourse/pipelines/pipeline.yml#L677

File: concourse/pipelines/pr_pipeline.yml [hunk 2]
```diff
@@ -124,6 +142,18 @@ jobs:
         timeout: 2h30m
         on_failure: *pr_failure
 
+      - task: icw_gporca_with_conan_ubuntu16
```
Comment (kmacoskey) [inline concourse/pipelines/pr_pipeline.yml]: Should this be renamed to match the same job in the other pipelines?
`icw_gporca_conan_ubuntu16`
https://github.com/greenplum-db/gpdb/blob/master/concourse/pipelines/pipeline.yml#L1290

File: concourse/pipelines/pr_pipeline.yml (hunk 1 above)
Comment (goutamtadi1) [reply concourse/pipelines/pr_pipeline.yml]: ✅

File: concourse/pipelines/pr_pipeline.yml (hunk 2 above)
Comment (goutamtadi1) [reply concourse/pipelines/pr_pipeline.yml]: ✅

**Pull request merged**